modules/exploits/windows/local/ms16_075_reflection_juicy.rb

```diff
@@ -21,6 +21,7 @@ def initialize(info={})
         This module utilizes the Net-NTLMv2 reflection between DCOM/RPC
         to achieve a SYSTEM handle for elevation of privilege.
         It requires a CLSID string.
+        Windows 10 > version 1803, build 17134 are not vulnerable.
```

Comment:
How about this, as per your code comments:

```suggestion
        Windows 10 and Server 2019, after build 17134 (April 2018 update, version 1803),
        are not vulnerable.
```

Reply:
I pushed a slight revision on this wording, to make it clear that _all_ versions of Server 2019 are not vulnerable. Thanks for your feedback!